bumble/avdtp.py

```diff
@@ -2162,6 +2162,9 @@ def on_close_command(self):
     def on_abort_command(self):
         self.emit('abort')
 
+    def on_delayreport_command(self, delay):
+        self.emit('delay_report', delay)
```

Comment:
Though this is accessed by reflection, could you add type annotation to help user understand the type of emitted value?

Reply:
done.